src/js/controller/model.js

```diff
@@ -260,7 +260,12 @@ define([
             return this[attr];
         },
         'set' : function(attr, val) {
+            if (this[attr] === val) {
+                return;
+            }
             this[attr] = val;
+            this.trigger('change:'+attr, this, val);
+            this.trigger('change', this);
```

Comment:
why do we need both `change:x` and `change`? won't `on('change')` act as a catch all?